static/bidder-info/openweb.yaml

```diff
@@ -0,0 +1,12 @@
+maintainer:
+  email: "monetization@openweb.com"
+gvlVendorID: 280
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
```

Comment:
We've made changes to how user syncs are configured since you first opened this PR. If OpenWeb supports user syncing and doesn't have a good default, please add the following to the end of this yaml file. Replace 'redirect' with 'iframe' if appropriate.

```
userSync:
  # openweb supports user syncing, but requires configuration by the host. contact this
  # bidder directly at the email address in this file to ask about enabling user sync.
  supports:
    - redirect
```